packages/deployment/test/test_lambda.js

```diff
@@ -87,7 +87,42 @@ test.serial('zipLambda: works for lambda using message adapter', async (t) => {
 });
 
 test.serial(
```

Comment:
Is there a reason that these tests can't be run in parallel?

Reply:
Hmm.  Probably not, the setup function tempdir should be unique, it doesn't appear anything here should collide, assuming I'm not missing anything in the underlying dependencies.... why was it implemented that way initially  🤔

Reply:
Running them in parallel results in several failures.   I can dig into that further and put up another PR if appropriate, or address it here if it feels like it should be sorted in this PR.

Reply:
Sounds like it's a defensive thing due to localstack constraints as much as anything - see https://github.com/cumulus-nasa/cumulus/pull/416